Sources/SimpleCurrencyConverter/CurrencyConverterProtocol.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+public protocol CurrencyConverterProtocol {
+    
+    /// This function is used to retrieve the exchange rate for a given base currency and a list of target currencies.
+    ///
+    /// This function uses the network manager to make a request to an exchange rates API, passing the base currency code and the target currency codes as parameters.
+    /// If the request is successful, the returned exchange rates are passed to the resolve closure.
+    /// If the request fails, the error is passed to the reject closure.
+    ///
+    ///  - Parameter baseCurrency: A Currency object representing the base currency for which the exchange rate is desired.
+    ///  - Parameter targetCurrencies: An array of Currency objects representing the target currencies for which the exchange rates are desired.
+    ///  - Parameter resolve: A closure that takes in a dictionary of exchange rates, where the keys are the currency codes of the target currencies and the values are the exchange rates (expressed as a Double)
```

Comment:
Wouldn't it be more consistent if you passed `Currency` type instead of `String` in `resolve`?

Reply:
Yes, it can be in that case.